packages/ketcher-react/src/script/editor/tool/bond.ts

```diff
@@ -303,18 +303,24 @@ class BondTool implements Tool {
   mouseup(event) {
     if ('dragCtx' in this) {
       const dragCtx = this.dragCtx
-      const rnd = this.editor.render
-      const struct = rnd.ctab.molecule
+      const render = this.editor.render
+      const struct = render.ctab.molecule
       if ('action' in dragCtx) {
         this.restoreBondWhenHoveringOnCanvas(event)
         this.editor.update(dragCtx.action)
       } else if (!('item' in dragCtx)) {
-        const xy = rnd.page2obj(event)
+        const editorOptions = this.editor.options()
+        const QUARTER_OF_BOND_WIDTH = 20 * editorOptions.zoom
```

Comment:
Would be better if we had some constant instead of `20` magic number. Or I think, it is supposed to be calculated somehow